sdk/README.md

```diff
@@ -57,10 +78,31 @@ Sample code for the application which receive data from others.
 Source code: samples/sender_app.c
 ```bash
 $ ./build/samples/recver_app
-usage: recver_app [OPTION]
--h, --help                      Print this help and exit.
--r, --ip=ip_address             Receive data from IP address (defaults: 127.0.0.1).
--p, --port=port_number  Receive data from Port (defaults: 9001).
+Usage: recver_app [OPTION]
+-H, --help                              Print this help and exit
+-r, --rcv_ip=<ip_address>               Receiver's IP address (default: 127.0.0.1)
+-i, --rcv_port=<port_number>            Receiver's port number (default: 9001)
+-s, --send_ip=<ip_address>              Sender's IP address (default: 127.0.0.1)
+-p, --send_port=<port_number>           Sender's port number (default: 9001)
+-o, --protocol=<protocol_type>          Set protocol type (default: auto)
+-t, --type=<payload_type>               Payload type (default: st20)
+-k, --socketpath=<socket_path>          Set memif socket path (default: /run/mcm/mcm_rx_memif.sock)
+-m, --master=<is_master>                Set memif conn is master (default: 1 for sender, 0 for recver)
+-d, --interfaceid=<interface_id>        Set memif conn interface id (default: 0)
+-b, --dumpfile=<file_name>              Save stream to local file (example: data-sdk.264)
+--------------------------------------   VIDEO (ST2x)   --------------------------------------
+-w, --width=<frame_width>               Width of test video frame (default: 1920)
+-h, --height=<frame_height>             Height of test video frame (default: 1080)
+-f, --fps=<video_fps>                   Test video FPS (frame per second) (default: 30.00)
+-x, --pix_fmt=<pixel_format>            Pixel format (default: yuv422p10le)
+--------------------------------------   AUDIO (ST3x)   --------------------------------------
+-a, --audio_type=<audio_type>           Define audio type [frame|rtp] (default: frame)
+-j, --audio_format=<audio_format>       Define audio format [pcm8|pcm16|pcm24|am824] (default: pcm16)
```

Comment:
I guess, pcm24 is supposed to be default

Reply:
Good catch!
```c
#define DEFAULT_AUDIO_FORMAT "pcm16"
```

Reply:
Quickly double-checked on the README. As the whole section was copied directly from runs of `./sender_app -h` and `./recver_app -h` where those values are printed from the defaults' definitions, all of them are fine.